python/vowpalwabbit/pyvw.py

```diff
@@ -1157,6 +1157,47 @@ def __str__(self):
             ["{}:{}:{}".format(c.action, c.cost, c.probability) for c in self.costs]
         )
 
+class ccb_label(abstract_label):
+    """Class for conditional contextual bandits VW label"""
+
+    def __init__(self, type = "unset", cost=0, top_class=0, top_probability=0, explicitly_included_actions=[]):
+        abstract_label.__init__(self)
+        if isinstance(cost, example):
```

Comment:
Should be `type` as that is the first arg no?
```suggestion
        if isinstance(type, example):
```